AI Techwriter documentation update - 2024-12-02 (#24)
PedroMiolaSilva/docs: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx

## Step 1. Enable Origin Shield

To activate Origin Shield, follow these steps:

- Ensure that the [Network Layer Protection module](/en/documentation/products/guides/secure/edge-firewall-configure-main-settings/#modules) is enabled in your account.
- Subscribe to the Origin Shield add-on by contacting the [sales team](https://www.azion.com/en/contact-sales/).

> - Once subscribed, access the Azion Console or Real-Time Manager to activate the feature. [TECHWRITER TODO: INSERT INFORMATION ABOUT HOW TO ACTIVATE ORIGIN SHIELD IN AZION CONSOLE/RTM]

Reviewer: ```suggestion
- Once subscribed, access the Azion Console. On more information on how to subscribe or activate this add-on, [read the documentation](/en/documentation/products/secure/edge-firewall/network-layer-protection/#origin-shield)
```
